### ISSUE TITLE
使用splang启动服务模型加载卡在0%

### ISSUE THREAD

**Issue by freeman745** [allenai/olmocr#200]
### 🐛 Describe the bug

根据readme运行pdf的解析，加载已经下载好的本地目录权重一直卡在0%
```
processor = AutoProcessor.from_pretrained(model_path, trust_remote_code=True)
model = AutoModelForVision2Seq.from_pretrained(model_path, trust_remote_code=True, device_map="auto")
model.eval()
```
这样是可以正常加载模型的，是什么原因？

### Versions

root@docker-desktop:/workspace/olmocr# python --version && pip freeze
Python 3.11.10
accelerate==1.7.0
aiohappyeyeballs==2.6.1
aiohttp==3.11.18
aiosignal==1.3.2
annotated-types==0.7.0
anthropic==0.51.0
anyio==4.9.0
archspec @ file:///home/conda/feedstock_root/build_artifacts/archspec_1708969572489/work
asttokens @ file:///home/conda/feedstock_root/build_artifacts/asttokens_1698341106958/work
astunparse==1.6.3
attrs @ file:///home/conda/feedstock_root/build_artifacts/attrs_1722977137225/work
beaker-py==2.3.0
beautifulsoup4 @ file:///home/conda/feedstock_root/build_artifacts/beautifulsoup4_1705564648255/work
bleach==6.2.0
boltons @ file:///home/conda/feedstock_root/build_artifacts/boltons_1711936407380/work
boto3==1.38.17
botocore==1.38.17
Brotli @ file:///home/conda/feedstock_root/build_artifacts/brotli-split_1725267488082/work
cached_path==1.7.3
cachetools==5.5.2
certifi @ file:///home/conda/feedstock_root/build_artifacts/certifi_1725278078093/work/certifi
cffi @ file:///home/conda/feedstock_root/build_artifacts/cffi_1725560564262/work
chardet @ file:///home/conda/feedstock_root/build_artifacts/chardet_1724954797915/work
charset-normalizer @ file:///home/conda/feedstock_root/build_artifacts/charset-normalizer_1728479282467/work
click @ file:///home/conda/feedstock_root/build_artifacts/click_1692311806742/work
cloudpickle==3.1.1
colorama @ file:///home/conda/feedstock_root/build_artifacts/colorama_1666700638685/work
compressed-tensors==0.8.0
conda @ file:///home/conda/feedstock_root/build_artifacts/conda_1729155142459/work
conda-build @ file:///home/conda/feedstock_root/build_artifacts/conda-build_1727204250023/work
conda-libmamba-solver @ file:///home/conda/feedstock_root/build_artifacts/conda-libmamba-solver_1727359833193/work/src
conda-package-handling @ file:///home/conda/feedstock_root/build_artifacts/conda-package-handling_1729006966809/work
conda_index @ file:///home/conda/feedstock_root/build_artifacts/conda-index_1718383105992/work
conda_package_streaming @ file:///home/conda/feedstock_root/build_artifacts/conda-package-streaming_1729004031731/work
cryptography==44.0.3
cuda-bindings==12.9.0
cuda-python==12.9.0
datasets==3.6.0
decorator @ file:///home/conda/feedstock_root/build_artifacts/decorator_1641555617451/work
decord==0.6.0
Deprecated==1.2.18
dill==0.3.8
diskcache==5.6.3
distro @ file:///home/conda/feedstock_root/build_artifacts/distro_1704321475663/work
dnspython==2.7.0
einops==0.8.1
exceptiongroup @ file:///home/conda/feedstock_root/build_artifacts/exceptiongroup_1720869315914/work
executing @ file:///home/conda/feedstock_root/build_artifacts/executing_1725214404607/work
expecttest==0.2.1
fastapi==0.115.12
filelock @ file:///home/conda/feedstock_root/build_artifacts/filelock_1726613473834/work
flashinfer==0.1.6+cu124torch2.4
frozendict @ file:///home/conda/feedstock_root/build_artifacts/frozendict_1728841334936/work
frozenlist==1.6.0
fsspec==2024.10.0
ftfy==6.3.1
gguf==0.10.0
google-api-core==2.24.2
google-auth==2.40.1
google-cloud-core==2.4.3
google-cloud-storage==2.19.0
google-crc32c==1.7.1
google-resumable-media==2.7.2
googleapis-common-protos==1.70.0
grpcio==1.71.0
h11==0.16.0
h2 @ file:///home/conda/feedstock_root/build_artifacts/h2_1634280454336/work
hf_transfer==0.1.9
hpack==4.0.0
httpcore==1.0.9
httptools==0.6.4
httpx==0.28.1
huggingface-hub==0.31.2
hyperframe @ file:///home/conda/feedstock_root/build_artifacts/hyperframe_1619110129307/work
hypothesis==6.115.5
idna @ file:///home/conda/feedstock_root/build_artifacts/idna_1726459485162/work
img2pdf==0.6.1
importlib_metadata==8.7.0
importlib_resources @ file:///home/conda/feedstock_root/build_artifacts/importlib_resources_1725921340658/work
interegular==0.3.3
ipython @ file:///home/conda/feedstock_root/build_artifacts/ipython_1729866374957/work
jedi @ file:///home/conda/feedstock_root/build_artifacts/jedi_1696326070614/work
Jinja2 @ file:///home/conda/feedstock_root/build_artifacts/jinja2_1715127149914/work
jiter==0.9.0
jmespath==1.0.1
jsonpatch @ file:///home/conda/feedstock_root/build_artifacts/jsonpatch_1695536281965/work
jsonpointer @ file:///home/conda/feedstock_root/build_artifacts/jsonpointer_1725302941992/work
jsonschema @ file:///home/conda/feedstock_root/build_artifacts/jsonschema_1720529478715/work
jsonschema-specifications @ file:///tmp/tmpvslgxhz5/src
lark==1.2.2
libarchive-c @ file:///home/conda/feedstock_root/build_artifacts/python-libarchive-c_1725302626023/work
libmambapy @ file:///home/conda/feedstock_root/build_artifacts/mamba-split_1727883551957/work/libmambapy
lief @ file:///home/conda/feedstock_root/build_artifacts/lief_1726040283347/work/api/python
lingua-language-detector==2.1.0
lintrunner==0.12.5
litellm==1.69.3
llvmlite==0.44.0
lm-format-enforcer==0.10.11
lxml==5.4.0
mamba @ file:///home/conda/feedstock_root/build_artifacts/mamba-split_1727883551957/work/mamba
markdown-it-py==3.0.0
markdown2==2.5.3
MarkupSafe @ file:///home/conda/feedstock_root/build_artifacts/markupsafe_1729351299924/work
matplotlib-inline @ file:///home/conda/feedstock_root/build_artifacts/matplotlib-inline_1713250518406/work
mdurl==0.1.2
menuinst @ file:///home/conda/feedstock_root/build_artifacts/menuinst_1725359023560/work
mistral_common==1.5.4
modelscope==1.26.0
more-itertools @ file:///home/conda/feedstock_root/build_artifacts/more-itertools_1725630081114/work
mpmath==1.3.0
msgpack==1.1.0
msgspec==0.19.0
multidict==6.4.3
multiprocess==0.70.16
nest-asyncio==1.6.0
networkx==3.4.2
ninja==1.11.1.1
numba==0.61.2
numpy==1.26.4
nvidia-cublas-cu12==12.4.5.8
nvidia-cuda-cupti-cu12==12.4.127
nvidia-cuda-nvrtc-cu12==12.4.127
nvidia-cuda-runtime-cu12==12.4.127
nvidia-cudnn-cu12==9.1.0.70
nvidia-cufft-cu12==11.2.1.3
nvidia-curand-cu12==10.3.5.147
nvidia-cusolver-cu12==11.6.1.9
nvidia-cusparse-cu12==12.3.1.170
nvidia-ml-py==12.575.51
nvidia-nccl-cu12==2.21.5
nvidia-nvjitlink-cu12==12.4.127
nvidia-nvtx-cu12==12.4.127
-e git+https://github.com/allenai/olmocr.git@66f9b468696485391d8a0a5d43c596c1da2b82f2#egg=olmocr
openai==1.75.0
opencv-python-headless==4.11.0.86
optree==0.13.0
orjson==3.10.18
outlines==0.0.46
packaging @ file:///home/conda/feedstock_root/build_artifacts/packaging_1718189413536/work
pandas==2.2.3
parso @ file:///home/conda/feedstock_root/build_artifacts/parso_1712320355065/work
partial-json-parser==0.2.1.1.post5
pexpect @ file:///home/conda/feedstock_root/build_artifacts/pexpect_1706113125309/work
pickleshare @ file:///home/conda/feedstock_root/build_artifacts/pickleshare_1602536217715/work
pikepdf==9.7.0
pillow==11.2.1
pkginfo @ file:///home/conda/feedstock_root/build_artifacts/pkginfo_1728616516923/work
pkgutil_resolve_name @ file:///home/conda/feedstock_root/build_artifacts/pkgutil-resolve-name_1694617248815/work
platformdirs @ file:///home/conda/feedstock_root/build_artifacts/platformdirs_1726613481435/work
pluggy @ file:///home/conda/feedstock_root/build_artifacts/pluggy_1713667077545/work
prometheus-fastapi-instrumentator==7.1.0
prometheus_client==0.21.1
prompt_toolkit @ file:///home/conda/feedstock_root/build_artifacts/prompt-toolkit_1727341649933/work
propcache==0.3.1
proto-plus==1.26.1
protobuf==5.29.4
psutil @ file:///home/conda/feedstock_root/build_artifacts/psutil_1729847065645/work
ptyprocess @ file:///home/conda/feedstock_root/build_artifacts/ptyprocess_1609419310487/work/dist/ptyprocess-0.7.0-py2.py3-none-any.whl
pure_eval @ file:///home/conda/feedstock_root/build_artifacts/pure_eval_1721585709575/work
py-cpuinfo==9.0.0
pyairports==2.1.1
pyarrow==20.0.0
pyasn1==0.6.1
pyasn1_modules==0.4.2
pycosat @ file:///home/conda/feedstock_root/build_artifacts/pycosat_1696355758146/work
pycountry==24.6.1
pycparser @ file:///home/conda/feedstock_root/build_artifacts/pycparser_1711811537435/work
pydantic==2.11.4
pydantic_core==2.33.2
Pygments @ file:///home/conda/feedstock_root/build_artifacts/pygments_1714846767233/work
pypdf==5.5.0
pypdfium2==4.30.1
PySocks @ file:///home/conda/feedstock_root/build_artifacts/pysocks_1661604839144/work
python-dateutil==2.9.0.post0
python-dotenv==1.1.0
python-etcd==0.4.5
python-multipart==0.0.20
pytz @ file:///home/conda/feedstock_root/build_artifacts/pytz_1726055524169/work
PyYAML @ file:///home/conda/feedstock_root/build_artifacts/pyyaml_1725456139051/work
pyzmq==26.4.0
ray==2.46.0
referencing @ file:///home/conda/feedstock_root/build_artifacts/referencing_1714619483868/work
regex==2024.11.6
requests @ file:///home/conda/feedstock_root/build_artifacts/requests_1717057054362/work
rich==13.9.4
rpds-py @ file:///home/conda/feedstock_root/build_artifacts/rpds-py_1725327039958/work
rsa==4.9.1
ruamel.yaml @ file:///home/conda/feedstock_root/build_artifacts/ruamel.yaml_1728764976564/work
ruamel.yaml.clib @ file:///home/conda/feedstock_root/build_artifacts/ruamel.yaml.clib_1728724459810/work
s3transfer==0.12.0
safetensors==0.5.3
sentencepiece==0.2.0
setproctitle==1.3.6
sgl-kernel==0.0.3.post1
sglang==0.4.2
six @ file:///home/conda/feedstock_root/build_artifacts/six_1620240208055/work
smart-open==7.1.0
sniffio==1.3.1
sortedcontainers==2.4.0
soupsieve @ file:///home/conda/feedstock_root/build_artifacts/soupsieve_1693929250441/work
stack-data @ file:///home/conda/feedstock_root/build_artifacts/stack_data_1669632077133/work
starlette==0.46.2
sympy==1.13.1
tiktoken==0.9.0
tokenizers==0.20.3
torch==2.5.1+cu124
torchao==0.11.0
torchaudio==2.5.1+cu124
torchelastic==0.2.2
torchvision==0.20.1+cu124
tqdm @ file:///home/conda/feedstock_root/build_artifacts/tqdm_1722737464726/work
traitlets @ file:///home/conda/feedstock_root/build_artifacts/traitlets_1713535121073/work
transformers==4.46.2
triton==3.1.0
truststore @ file:///home/conda/feedstock_root/build_artifacts/truststore_1724770958874/work
types-dataclasses==0.6.6
typing-inspection==0.4.0
typing_extensions @ file:///home/conda/feedstock_root/build_artifacts/typing_extensions_1717802530399/work
tzdata==2025.2
urllib3 @ file:///home/conda/feedstock_root/build_artifacts/urllib3_1726496430923/work
uvicorn==0.34.2
uvloop==0.21.0
vllm==0.6.4.post1
watchfiles==1.0.5
wcwidth @ file:///home/conda/feedstock_root/build_artifacts/wcwidth_1704731205417/work
webencodings==0.5.1
websockets==15.0.1
wrapt==1.17.2
xformers==0.0.28.post3
xgrammar==0.1.19
xxhash==3.5.0
yarl==1.20.0
zipp @ file:///home/conda/feedstock_root/build_artifacts/zipp_1726248574750/work
zstandard==0.23.0

**Comment by freeman745** [comment]
发现问题是加载模型超级慢
用automodel加载需要6-7分钟
用splang需要20分钟以上
